scripts/fonts/add-polish-glyphs.py

```
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+import fontforge, psMat, os
+
+f = fontforge.open("/tmp/fontpatch/orig/ChicagoFLF.woff")
```

Ruff (0.15.17)

[error] Probable insecure usage of temporary file or directory: "/tmp/fontpatch/orig/ChicagoFLF.woff"

(S108)